Sources/Refresher/Refresher.swift

```diff
@@ -161,7 +162,7 @@ public struct RefreshableScrollView<Content: View, RefreshView: View>: View {
     
     @ViewBuilder
     private var refershSpinner: some View {
-        if showRefreshControls && (state.style == .default || state.style == .overlay) {
+        if style == .default || style == .overlay {
```

Comment:
https://github.com/gh123man/SwiftUI-Refresher/blob/3aa4d74f2c71eafc540bd29ca66bfac684d385fb/Sources/Refresher/Refresher.swift#L257

This condition was causing it to trigger consistently when scrolling the view downward, leading to the unnecessary redraw of the RefreshSpinner.